### DISCUSSION TITLE
[Feature Request] User Management Portal

### DISCUSSION THREAD

**Discussion by ados8** [donetick/donetick#50] [Ideas]
I've just begun testing this app to see if I can use it with the home and family.
One thing I noticed right away is no user portal, will this be something on the horizon.
I think it's important for an admin to be able to manage any account that they are hosting, restrict and prevent sign-ups.
This would also eliminate the need to use an SMTP server as the admin can reset passwords for user.
Thank you for developing this app I will continue to review.

Edit:
For now I will block the `/signup` URL and place behind my SSO server for security.

**Comment by meauxt** [comment]
Hi @ados8!  Gonna move this to discussion. right now there is a way to disable signup but user management something not sure if am planning to work on in the meanwhile. maybe in future!

**Reply by ados8** [reply]
Any updates on this? Be great to have a user portal where you can see all accounts, reset passwords and delete users.

**Comment by ados8** [comment]
@meauxt bummer, in many cases the tech person in the family manages everything for the non tech people.
This means resetting accounts, details etc. Was hoping for an admin backend to manage accounts.

**Comment by hales8181** [comment]
Same, having to sign up each individual user and join the circle then approve them felt a bit long winded - an admin portal where you could add & manage users instead would be really good.  Also the ability to make the other parent an admin and only the parent accounts have the ability to add and assign tasks
